cms/envs/common.py

```diff
@@ -2326,13 +2326,13 @@
 # .. toggle_name: ERROR_ON_DEPRECATED_EDX_PLATFORM_IMPORTS
 # .. toggle_implementation: DjangoSetting
 # .. toggle_default: False
-# .. toggle_use_cases: rollout
+# .. toggle_use_cases: temporary
 # .. toggle_creation_date: 2021-01-20
 # .. toggle_target_removal_date: 2021-01-27
 # .. toggle_tickets: https://github.com/edx/edx-platform/pull/25932
 # .. toggle_description: Whether to raise an exception where,
-#  normally, a DeprecatedEdxPlatformImportWarning would be raised.
-#  This will allow us to test dropping support for the deprecated
-#  import paths without yet removing all of the import_shims
-#  machinery.
+#   normally, a DeprecatedEdxPlatformImportWarning would be raised.
```

Comment:
@regisb: I’ve seen this elsewhere, and wondering if it might be a common issue in the future. As a lower priority fix, would we be able to simply make this an acceptable multi-line indent?

Reply:
> As a lower priority fix, would we be able to simply make this an acceptable multi-line indent?

No, not really. At least, I don't think so. The issue comes from the regexp that parses the multi-line comments. I suspect that if we reduce the minimal number of required additional empty spaces we will stumble on backward incompatibilities. I'm not quite sure, though.

Reply:
Since linting should catch this, I say we forget about it.  Thanks.

Reply:
> Since linting should catch this

Actually, this is not the case. If the description is incorrectly indented, then it will simply not be parsed, and the linter will fail to see the remaining of the description. The best the linter could do is to check whether the description is only line long and does end with a "." dot.

Reply:
Thanks.  I took notes on this for a possible improvement. We can discuss at a later time.  Thanks.